recipes/cython_gsl/meta.yaml

```diff
@@ -1,19 +1,21 @@
-{% set name = "CythonGSL" %}
+{% set name = "cython_gsl" %}
```

Comment:
The original name is `cythongsl` isn't it? No need to add the underscore. (I know that CamelCase names converted to  lower case looks weird though.)